ls1intum/Artemis
# View FileUpload / Modeling / Text submission from Exercise Scores page

### Checklist

### Motivation and Context
This PR addresses issue #1120 which states that it is not possible to reach the submission itself for TextExercise and FileUploadExercise exercises from the scores or participations page (i.e. not reachable via any link or button, only by manually extracting the submission id and inserting it into a url pattern in the browser url bar). For programming exercises this is possible by clicking on the "Repository" button or link.

### Description
Note: You need to be an instructor for that exercise.
1. Add button "View submission" to scores page for FileUploadExercise (opens the assessment page for the latest file upload submission)
2. Add button "View submission" to scores page for TextExercise (opens the assessment page for the latest text submission)
3. Add button "View submission" to scores page for ModelingExercise (opens the assessment page for the latest modeling submission)

### Steps for Testing
1. Log in to Artemis as an instructor
2. Navigate to Course Administration
3. Click on "Exercises" for a course with a file upload, modeling and text exercise
4. Click on "Scores" for a file upload, modeling and text exercise
5. Check that the new button (either "Download file" or "View Text Submission" or "View Modeling Submission") is present and performs its function as described above


### Screenshots
Note: Button text has changed to "View submission".

![OnPaste 20191215-143754](https://user-images.githubusercontent.com/7109225/70863452-93713e00-1f48-11ea-8b61-50853ca178bb.png)
![OnPaste 20191215-144113](https://user-images.githubusercontent.com/7109225/70863481-f662d500-1f48-11ea-9a44-ae7948ca6af5.png)
![OnPaste 20191215-153608](https://user-images.githubusercontent.com/7109225/70864245-9ff99480-1f50-11ea-9e42-1d1d6535ec7b.png)
